configure.ac

```diff
@@ -633,6 +641,29 @@ if test "${enable_pcsc}" = "yes"; then
 	AC_DEFINE([ENABLE_PCSC], [1], [Define if PC/SC is to be enabled])
 fi
 
+if test "${enable_cryptotokenkit}" = "yes"; then
+	if test -z "${CRYPTOTOKENKIT_CFLAGS}"; then
+		case "${host}" in
+			*-apple-*)
+				# Locate the latest SDK.
+				SDKS_PATH="$(xcode-select --print-path)/Platforms/MacOSX.platform/Developer/SDKs"
+				if test -d $SDKS_PATH; then
+					SDK_PATH="${SDK_PATH:-$SDKS_PATH/$(ls -1 ${SDKS_PATH} | sort -n -t. -k2 -r | head -1)}"
+					CRYPTOTOKENKIT_CFLAGS="-I$SDK_PATH/System/Library/Frameworks/CryptoTokenKit.framework/Versions/Current/Headers"
+				else
+					CRYPTOTOKENKIT_CFLAGS="-I/System/Library/Frameworks/CryptoTokenKit.framework/Headers"
```

Comment:
And if needed we can use -isysroot ${SDK_PATH}.
OSX compilator can figure out all the paths with frameworks for include and link parameters